index.yaml

```diff
@@ -16,5 +16,8 @@ indexes:
 - kind: TestRun
   properties:
   - name: BrowserName
+  - name: BrowserVersion
+  - name: OSName
+  - name: OSVersion
```

Comment:
Unfortunately I think we need 3 separate indexes now. I'm pretty sure you need an index for each unique set of terms you want to query :cry:

Reply:
Done - do we have any concerns for how large these indexes could get?